backend/src/models/skill.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseModel, Field
+import uuid
+
+
+class SkillInterface(BaseModel):
```

Comment:
should this be just called `Skill`?

Reply:
Why do we need this interface definition if you were able to add metadata into the tool class itself?